app/uk/gov/nationalarchives/omega/editorial/services/jms/StubServer.scala

```diff
@@ -81,8 +82,25 @@ class StubServer @Inject() (responseBuilder: ResponseBuilder) {
       requestMessageId <- responseBuilder.jmsMessageId(jmsMessage)
       _                <- logger.info(s"got a message with ID $requestMessageId")
       responseText     <- responseBuilder.createResponseText(jmsMessage)
-      responseMessage  <- messageFactory.makeTextMessage(responseText)
-      _ = responseMessage.setJMSCorrelationId(requestMessageId)
-    } yield responseMessage
+      replyMessage     <- messageFactory.makeTextMessage(responseText)
+      _ = replyMessage.setJMSCorrelationId(requestMessageId)
+      _ = replyMessage.setStringProperty(MessageProperties.OMGApplicationID, "PACS001")
+      _ = replyMessage.setStringProperty(
+            MessageProperties.OMGMessageTypeID,
+            getReplyMessageType(jmsMessage.getStringProperty(MessageProperties.OMGMessageTypeID))
+          )
+      _ = replyMessage.setStringProperty(MessageProperties.OMGMessageFormat, "application/json")
+      _ = replyMessage.setStringProperty(MessageProperties.OMGReplyAddress, "PACS001.request")
+      _ = replyMessage.setStringProperty(MessageProperties.OMGToken, "AbCdEf123456")
+    } yield replyMessage
+
+  private val messageTypeMap = Map[String, String]("OSLISALS001" -> "ODLISALS001")
+  private def getReplyMessageType(maybeMessageType: Option[String]): String = {
```

Comment:
why does the reply `MessageTypeId` change here?

Reply:
There is a different `MessageTypeId` for the request and reply so in this case `OSLISALS001` is for the request and `ODLISALS001` is for the reply. This information is used to indicate how to unmarshall the reply payload. We are not doing anything with this information at the moment but I wanted it to be as realistic as possible for when need this.